curvefs/src/client/fuse_client.cpp

```diff
@@ -558,13 +556,20 @@ CURVEFS_ERROR FuseClient::RemoveNode(fuse_req_t req, fuse_ino_t parent,
                    << ", inodeid = " << parent;
         return ret;
     }
-    ret = parentInodeWrapper->DecreaseNLink();
-    if (ret != CURVEFS_ERROR::OK) {
-        LOG(ERROR) << "inodeManager DecreaseNLink fail, ret = " << ret
-                   << ", inodeid = " << parent;
-        return ret;
+
+    {
+        curve::common::UniqueLock lk = parentInodeWrapper->GetUniqueLock();
+        struct timespec now;
+        clock_gettime(CLOCK_REALTIME, &now);
+        parentInodeWrapper->SetMTime(now.tv_sec, now.tv_nsec);
+        parentInodeWrapper->SetCTime(now.tv_sec, now.tv_nsec);
+
+        if (FsFileType::TYPE_DIRECTORY == type) {
+            parentInodeWrapper->InvalidateNlink();
+        }
```

Comment:
These operations alse need exec in 'FuseOpSymlink'  and 'FuseOpLink' and maybe can abstract a function in InodeWrapper.

Reply:
done